### DISCUSSION TITLE
can't find valid git repository. Should I init git manually?

### DISCUSSION THREAD

**Discussion by 5y1ee** [Vinzent03/obsidian-git#772] [Q&A] [answered]
I start Obsidian today, and made some valut with file. And I installed Obsidian Git plug-in and enabled it.
But It said, can't find valid git repository. please create one via the given command or clone an existing repo.
I searched issues and found check the command (ctrl+shift+i). But command said nothing.
I figured out that in my path of vault. I don't have .git folder. And i tried "git init" in command, Obsidian git recognize as a repo and works.

My curious is that, when I saw youtube and docs, there's not information that I should init git as manually. Am I missed it? Or it's just a simple process that didn't need to be described?

**Comment by Adasfans** [comment]
same problem。

**Comment by Vinzent03** [comment, marked as answer]
I've added now a small section on how to start with a new local repository. So yeah if you don't clone an existing remote repository, you have to init a local one first.
https://publish.obsidian.md/git-doc/Getting+Started#Create+new+local+repository

**Reply by Adasfans** [reply]
Thank you for your reply， I have solved this problem  three days ago 😊Friends who have the same problem, with your tutorial, should be able to solve this problem quickly, without spending a whole day like me.
